fatrace.c

```diff
@@ -461,8 +475,14 @@ print_event (const struct fanotify_event_metadata *data,
             putchar(',');
             print_json_str("path", pathname);
         }
+        if (option_exe == 1 && got_exepath) {
```

Comment:
Please treat `option_exe` like a boolean, just like in the other places. I.e. drop the `== 1`. Also feel free to actually *declare* it as a `bool`.